pkgconfig/shad_cpp_simple.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+exec_prefix=${prefix}
+includedir=${prefix}/include
+libdir=${exec_prefix}/lib
+
+Name: @CMAKE_PROJECT_NAME@_CPP_SIMPLE
+Description: SHAD - Scalable and High-Performance Algorithms and Data-Structures, CPP_SIMPLE Backend.
+Version: @PACKAGE_VERSION@
+Cflags: @CMAKE_CXX_FLAGS@ -D HAVE_CPP_SIMPLE=1 -I${includedir}
```

Comment:
it is better (and consistent with -I) if you remove the space between -D and HAVE_CPP_SIMPLE